### PR TITLE
Initial seed

### PR DESCRIPTION
**Pull request by tairezzzz**
Initial seed for bags-admin-client

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+node_js:
+  - '0.10'
+  - '0.12'
+  - stable
+
+env:
+  global:
+    - secure: CxEg2hSOUgwATFFAbD5ycalZdVvaKSTGIVyVYGze7NCqNaDhaXr1VSfrdfu+GrPp6+cYPhBB8Rr0LaweyZPqiVEHFWBtUiOjl5PShOazyiCNTX6ZQUCTHYQVkoIyKYhP/WVzmy6MfhCnuSkA129EeQ6dVUr/8NlIW2/N8PgU018=
+    - GH_OWNER: zckrs
+    - GH_PROJECT_NAME: gulp-angular-boilerplate
```
Comment (MicahZoltu) [inline .travis.yml:12]: Should these be changed to `Zoltu` and `bags-admin-client`?  Perhaps I am misunderstanding these variables?

Also, what variable(s) are in the encrypted section?  (not the values, just what is the type of data)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# How to contribute
```
Comment (MicahZoltu) [inline CONTRIBUTING.md:1]: This appears to be leftover from the template this was pulled from, can it be removed?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* perf: A code change that improves performance
+* test: Adding missing tests
+* chore: Changes to the build process or auxiliary tools and libraries such as documentation generation
+
+### Donations
+
+If you like this project, if you want a faster development, if you feel generous, donation is a great way to motivate us more.
+
+[![Gratipay](http://img.shields.io/gratipay/Swiip.svg?style=flat)](https://gratipay.com/Swiip/)
+[![Paypal](http://img.shields.io/badge/paypal-donate-yellow.svg?style=flat)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=FWGV3KKGH2D4S)
```
Comment (MicahZoltu) [inline CONTRIBUTING.md:65]: Even if we keep the file, the donations section should go away.

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Yeoman Gulp-Angular Generator documentation
```
Comment (MicahZoltu) [inline docs/README.md:1]: It looks like this file is also leftover from the original template, can it be deleted?  I suspect the entire `docs` folder can actually be deleted.

File: generators/app/advanced-prompts.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+[
```
Comment (MicahZoltu) [inline generators/app/advanced-prompts.json:1]: I'm guessing the `generators` folder can also be removed?

**Comment by MicahZoltu** [comment]
Consider cleaning out any un-used files from the repo to make it easier for future to figure out what files to look at.  However, that can be done in a future PR or after the project has developed a bit.

File: .travis.yml (hunk 1 above)
Comment (tairezzzz) [inline .travis.yml:12]: I'm using this yeoman project https://github.com/Swiip/generator-gulp-angular to develop such kind of apps. This file was needed only for initiating seed with desired techs. I'm removing it now.

File: CONTRIBUTING.md (hunk 2 above)
Comment (tairezzzz) [inline CONTRIBUTING.md:1]: This file will be removed.

File: CONTRIBUTING.md (hunk 3 above)
Comment (tairezzzz) [inline CONTRIBUTING.md:65]: This file will be removed.

File: docs/README.md (hunk 4 above)
Comment (tairezzzz) [inline docs/README.md:1]: `docs` dir is using for initial seed documentation, we can remove it with ease

File: generators/app/advanced-prompts.json (hunk 5 above)
Comment (tairezzzz) [inline generators/app/advanced-prompts.json:1]: Yeah we can do that, but should remove `scripts` dir too

**Pull request merged**